pkg/varlinkapi/containers.go

```diff
@@ -317,6 +318,9 @@ func (i *LibpodAPI) ExportContainer(call iopodman.VarlinkCall, name, outPath str
 
 // GetContainerStats ...
 func (i *LibpodAPI) GetContainerStats(call iopodman.VarlinkCall, name string) error {
+	if rootless.IsRootless() {
```

Comment:
This isn't global. It works with CGroups V2

Reply:
yeah but we dont have official support for that right?

Reply:
"github.com/containers/libpod/pkg/cgroups"

cgroupv2, _ := cgroups.IsCgroup2UnifiedMode()

if rootless.IsRootless() && ! cgroupv2

Reply:
We don't have official support for anything?  Including rootless.